ql/src/java/org/apache/hadoop/hive/ql/parse/EximUtil.java

```diff
@@ -237,6 +238,13 @@ public static void createDbExportDump(FileSystem fs, Path metadataPath, Database
     // If we later make this work for non-repl cases, analysis of this logic might become necessary. Also, this is using
     // Replv2 semantics, i.e. with listFiles laziness (no copy at export time)
 
+    // Remove all the entries from the parameters which are added for bootstrap dump progress
+    Map<String, String> parameters = dbObj.getParameters();
+    if (parameters != null) {
+      parameters.entrySet()
+                .removeIf(e -> e.getKey().startsWith(Utils.BOOTSTRAP_DUMP_STATE_KEY_PREFIX));
+      dbObj.setParameters(parameters);
+    }
```

Comment:
we have to reset the parameters object in the dbObj as the object is pass by reference and hence after the export of the properties from the object we might use it somewhere else where it might need access to original properties.

Reply:
ok